err.go

```diff
@@ -677,17 +677,14 @@ func WrapWC(ctx context.Context, err error, msg string) *Err {
 func Stack(errs ...error) *Err {
 	filtered := []error{}
 	for _, err := range errs {
-		val := reflect.ValueOf(err)
-
-		if err != nil &&
-			!((val.Kind() == reflect.Pointer || val.Kind() == reflect.Interface) &&
-				val.IsNil()) {
+		if !isNilErrIface(err) {
 			filtered = append(filtered, err)
 		}
 	}
 
 	switch len(filtered) {
 	case 0:
+		fmt.Printf("\n-----\nALL NIL %+v\n-----\n", filtered)
```

Comment:
nit: seems like a stray debug print that could be removed?
```suggestion
```

Reply:
d'oh!  Thanks